_guides/quickstart-app-development.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Quickstart for DHIS2 App Development
+category: guide
+layout: page
+---
+
+# Quickstart for DHIS2 App Development
+
+## Create a new app and connect it to a local DHIS2 instance in 10 minutes or less.
+
+### Introduction
+You can start developing a DHIS2 app on OSX, Windows or Linux using Docker and the DHIS command line interface (CLI). In this guide, you will install the prerequisites, start up a local DHIS2 instance and create a new app. Your app will connect to the local DHIS2 instance.
+
+### Installing prerequisites
+If you are using Debian Linux, 
+1. Follow the [DHIS2 Docker guide](https://developers.dhis2.org/guides/dhis2-docker) to install the prerequisites
+
+If you are using OSX or Windows,
+1. Install [Docker](https://docs.docker.com/get-docker/)
+3. Install [yarn](https://classic.yarnpkg.com/en/docs/install)
+4. From the command line or terminal, install DHIS CLI
+```
+yarn global add @dhis/cli
+```
+Now that you have installed Docker and the DHIS CLI you are ready to start up DHIS2
+
+### Starting a local DHIS2 instance
+
+1. From the terminal, start up DHIS2 and seed the database
+```
+d2 cluster up 2.35.0 --db-version 2.35 --seed
+```
+2. From the browser, navigate to http://localhost:8080. If DHIS2 is running, you should see the following page,
+![](../assets/quickstart_guides/image-of-login.png)
+
+If you can load DHIS in the browser, you are ready to create a new app
+
+### Creating a new app
+1. From the terminal, create a new DHIS2 app called "my-app"
+```
+d2 app scripts init my-app
+```
+2. Change directories to `/my-app` and start the app
+```
+cd my-app && yarn start
+```
+
+### Connecting your app to DHIS2
+1. From the browser, navigate to http://localhost:3000. You will see the following page
```

Comment:
And same thing here  ;-) 

`[http://localhost:3000](http://localhost:3000){:target="\_blank"}`